x-pack/plugins/synthetics/public/legacy_uptime/hooks/use_breadcrumbs.ts

```diff
@@ -57,7 +57,7 @@ export const makeBaseBreadcrumb = (
       href: observabilityPath,
     },
     {
-      text: i18n.translate('xpack.synthetics.breadcrumbs.overviewBreadcrumbText', {
+      text: i18n.translate('xpack.uptime.breadcrumbs.overviewBreadcrumbText', {
```

Comment:
I don't think this will pass ci. I think it'll need to be `xpack.synthetics`. No?